source/fundamentals/collations.txt

```diff
@@ -0,0 +1,531 @@
+
+   * - Alternate
+     - | Whether to consider spaces and punctuation.
+       | :java-docs:`collationAlternate() <apidocs/mongodb-driver-core/com/mongodb/client/model/Collation.Builder.html#collationAlternate(com.mongodb.client.model.CollationAlternate)>` API documentation
+
+   * - Case first
+     - | Whether to consider uppercase or lowercase first.
+       | :java-docs:`collationCaseFirst() <apidocs/mongodb-driver-core/com/mongodb/client/model/Collation.Builder.html#collationCaseFirst(com.mongodb.client.model.CollationCaseFirst)>` API documentation
+
+   * - Max Variable
+     - | Whether to ignore whitespace or both whitespace and punctuation. This setting is only valid when the alternate setting is "shifted".
+       | :java-docs:`collationMaxVariable() <apidocs/mongodb-driver-core/com/mongodb/client/model/Collation.Builder.html#collationMaxVariable(com.mongodb.client.model.CollationMaxVariable)>` API documentation
+
+   * - Strength
+     - | ICU level of comparison. The default value is "tertiary". For more information on each level, see the `ICU Comparison Levels <http://userguide.icu-project.org/collation/concepts#TOC-Comparison-Levels>`__.
+       | :java-docs:`collationStrength() <apidocs/mongodb-driver-core/com/mongodb/client/model/Collation.Builder.html#collationStrength(com.mongodb.client.model.CollationStrength)>` API documentation
+
+   * - Locale
+     - | **Required**. The ICU locale code for language and variant.
+       | :java-docs:`locale() <apidocs/mongodb-driver-core/com/mongodb/client/model/Collation.Builder.html#locale(java.lang.String)>` API documentation
+
+   * - Normalization
+     - | Whether to perform unicode normalization on the text as needed. For more information on unicode normalization, see `Unicode Normalization Forms <https://unicode.org/reports/tr15/>`__.
+       | :java-docs:`normalization() <apidocs/mongodb-driver-core/com/mongodb/client/model/Collation.Builder.html#normalization(java.lang.Boolean)>` API documentation
+
+   * - Numeric Ordering
+     - | Whether to order numbers according to numeric value rather than collation order.
+       | :java-docs:`numericOrdering() <apidocs/mongodb-driver-core/com/mongodb/client/model/Collation.Builder.html#numericOrdering(java.lang.Boolean)>` API documentation
+
+You can use the ``Collation.Builder`` class to specify values for the
+collation options above. You can call the ``build()`` method construct a
+``Collation`` object as shown in the following code snippet:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
+   :language: java
+   :dedent:
+   :start-after: start collationBuilder
+   :end-before: end collationBuilder
+
+For more information on the corresponding methods and parameters they
+take, see the API documentation for :java-docs:`Collation.Builder
```

Comment:
I don't think we've adopted any standard as both are used across Node and Java documentation. It could be a good topic for a future style discussion though.

Reply:
Added this as point 18 to the [style doc](https://docs.google.com/document/d/1SxlDvx4MYKBE6-LYGp5i5uuHfY1ZUYYlhc6syEdta5M/edit). I picked a stance on the issue for the sake of discussion, but am open to either (link on just the method name or link on the method name + context).